tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -2312,9 +3009,25 @@ function foo () {
 }
 ',
             ],
+            [
+                '<?php
```

Comment:
Test added for https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6716#issuecomment-1364486054